test/y2partitioner/widgets/md_devices_selector_test.rb

```diff
@@ -281,7 +283,7 @@
             expect(controller.devices_in_md.map(&:name)).to eq [
               "/dev/sde3", "/dev/sda3", "/dev/sda2", "/dev/sda4"
             ]
-            expect(controller.available_devices).to be_empty
+            expect(controller.available_devices.map(&:name)).to eq ["/dev/sdb", "/dev/sdc"]
```

Comment:
do we really want to be order dependent? For me `contain_exactly` is better unless we really want to check order.